runtime-modules/storage/src/tests/mocks.rs

```diff
@@ -2,62 +2,113 @@
 
 pub use frame_support::traits::LockIdentifier;
 use frame_support::weights::Weight;
-use frame_support::{ensure, impl_outer_event, impl_outer_origin, parameter_types};
+use frame_support::{
+    ensure, parameter_types,
+    traits::{ConstU16, ConstU32, ConstU64},
+    PalletId,
+};
 use frame_system::ensure_signed;
 use sp_core::H256;
 use sp_runtime::{
     testing::Header,
     traits::{BlakeTwo256, IdentityLookup},
-    DispatchError, DispatchResult, ModuleId, Perbill,
+    DispatchError, DispatchResult, Perbill,
+};
+use sp_std::{
+    cell::RefCell,
+    convert::{TryFrom, TryInto},
 };
-use sp_std::cell::RefCell;
 use staking_handler::LockComparator;
 
-// Workaround for https://github.com/rust-lang/rust/issues/26925 . Remove when sorted.
-#[derive(Clone, PartialEq, Eq, Debug)]
-pub struct Test;
-
-impl_outer_origin! {
-    pub enum Origin for Test {}
-}
-
-mod storage {
-    pub use crate::Event;
+parameter_types! {
+    pub const ExistentialDeposit: u32 = 1;
 }
-
-mod membership_mod {
-    pub use membership::Event;
+parameter_types! {
+    pub const BlockHashCount: u64 = 250;
+    pub const MaximumBlockWeight: u32 = 1024;
+    pub const MaximumBlockLength: u32 = 2 * 1024;
+    pub const AvailableBlockRatio: Perbill = Perbill::one();
+    pub const MinimumPeriod: u64 = 5;
 }
 
-impl_outer_event! {
-    pub enum TestEvent for Test {
-        balances<T>,
-        storage<T>,
-        frame_system<T>,
-        membership_mod<T>,
-        working_group Instance2 <T>,
-        working_group Instance9 <T>,
-    }
+// The storage working group instance alias.
+pub type StorageWorkingGroupInstance = working_group::Instance2;
+// The distribution working group instance alias.
+pub type DistributionWorkingGroupInstance = working_group::Instance3;
```

Comment:
In both `runtime/src/lib.rs` and `runtime-modules/utility/src/tests/mocks.ts` we have:
```suggestion
pub type DistributionWorkingGroupInstance = working_group::Instance9;
```

Reply:
They do not need to match as long as we are consistent in the specific constructed runtime.

Reply:
You might be right after all, as the benchmark command doesn't use the mock runtime, so it does make sense to use same instance number. Will update.